### PR TITLE
resynchronize eovsapy

### PR DESCRIPTION
**Pull request by sageyu123**

### CODE REVIEW

File: setup.py
Comment (sageyu123) [inline setup.py:1]: this file is replaced by pyproject.toml

File: setup.cfg
Comment (sageyu123) [inline setup.cfg:1]: this file is replaced by pyproject.toml

File: requirements.txt
Comment (sageyu123) [inline requirements.txt:1]: this file is replaced by pyproject.toml

File: eovsapy/shellScript/pipeline_plt_movie.sh
Comment (sageyu123) [inline eovsapy/shellScript/pipeline_plt_movie.sh:1]: /common/python/suncasa -> /common/python/suncasa-src/suncasa

File: eovsapy/shellScript/pipeline_plt.sh
Comment (sageyu123) [inline eovsapy/shellScript/pipeline_plt.sh:1]: /common/python/suncasa -> /common/python/suncasa-src/suncasa

File: eovsapy/shellScript/pipeline_compress.sh
Comment (sageyu123) [inline eovsapy/shellScript/pipeline_compress.sh:1]: /common/python/suncasa -> /common/python/suncasa-src/suncasa

File: eovsapy/shellScript/pipeline2.csh
Comment (sageyu123) [inline eovsapy/shellScript/pipeline2.csh:1]: /common/python/suncasa -> /common/python/suncasa-src/suncasa

File: eovsapy/shellScript/pipeline1.csh
Comment (sageyu123) [inline eovsapy/shellScript/pipeline1.csh:1]: /common/python/suncasa -> /common/python/suncasa-src/suncasa

File: eovsapy/shellScript/pipeline.sh
Comment (sageyu123) [inline eovsapy/shellScript/pipeline.sh:1]: /common/python/suncasa -> /common/python/suncasa-src/suncasa

File: eovsapy/read_idb.py
Comment (sageyu123) [inline eovsapy/read_idb.py:1]: please accept the one on the right.

File: eovsapy/flare_spec.py
Comment (sageyu123) [inline eovsapy/flare_spec.py:1]: Please review if the changes make sense to you. The updates are adding docstring and a new function to write spec data to fits.

File: eovsapy/autocorrect_tp.py
Comment (sageyu123) [inline eovsapy/autocorrect_tp.py:1]: The apparent differences between the code on the left and right are due to extra spaces after commas, hyphens, slashes, etc., which were automatically added by the coding software. There are no actual content differences between the two versions.

File: README.rst
Comment (sageyu123) [inline README.rst:1]: this file is replaced by the README.md

File: README.md
Comment (sageyu123) [inline README.md:1]: The file is added by Sijie.

File: pyproject.toml
Comment (sageyu123) [inline pyproject.toml:1]: The `pyproject.toml` file consolidates `setup.cfg`, `setup.py`, and `requirements.txt` into one standardized configuration file.

**Review by dgary50 — APPROVED** [review]

**Review by dgary50 — APPROVED** [review]

**Pull request closed without merging**